### ISSUE TITLE
Close more files explicitly

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8624]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8624|
|Type|enhancement|
|Created|2016-07-18 20:12:54Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8624 8624
type__enhancement enhancement
reporter__scop scop
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468872774499194 1468872774499194
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_1"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/351

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 64704ca9ddef828761b43b4161a87755fcc720d1
```
#!CommitTicketReference repository="" revision="64704ca9ddef828761b43b4161a87755fcc720d1"
Merge pull request #351 from scop/8624-scop-close

Author: scop
Reviewer: rodrigc
Fixes: #8624

Close more files explicitly
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
